manual/source/mmref-copyright.rst

```diff
@@ -1,26 +1,30 @@
 Copyright
 *********
 
-
-Use subject to copyright restrictions
-=====================================
-
 The copyright in The Memory Management Reference is owned by
 `Ravenbrook Limited`_.
 
 .. _Ravenbrook Limited: https://www.ravenbrook.com/
 
```

Comment:
Please update this to include the copyright notice in the same format as license.txt, that is
```
Copyright © 2001-2020 `Ravenbrook Limited <https://www.ravenbrook.com/>`_.
```